pt/scripts/iota_tracker.py

```diff
@@ -34,76 +34,154 @@
     "BEMS9PV9DISPLAY9": PV,
     "BEMS9WT9DISPLAY9": WT,
 }
+# Content of data_field in db
+DB_DATA_TYPE = ("Demand", "ESS", "EV", "PV", "WT")
 
 
-def process_data():
-    # get address from db
+def get_inserted_data(utc_now, db_data_type):
+    # Get inserted data of the hour
+    db_datas = [
+        (data.uuid, data.address)
+        for data in PowerData.query.filter(
+            PowerData.data_type == db_data_type,
+            extract("year", PowerData.updated_at) == utc_now.year,
+            extract("month", PowerData.updated_at) == utc_now.month,
+            extract("day", PowerData.updated_at) == utc_now.day,
+            extract("hour", PowerData.updated_at) == utc_now.hour,
+        ).all()
+    ]
+
+    if db_datas:
+        # Split uuid and address into two tuples
+        return map(tuple, zip(*db_datas))
+
+    # First query of the hour, no inserted data is normal
+    return [], []
```

Comment:
Maybe we need to add some comment in docstring to explain the meaning of first and last return object

Reply:
Added in 6c85155.